README.markdown

```diff
@@ -1,4 +1,5 @@
 # StarExec
+Author: Rahul Dass
```

Comment:
This is probably a mistake.

Reply:
```suggestion
Author: Rahul Dass
```

Reply:
Yes of course! You may recall that was a comment I added as part of the initial "test" branch to see if I (Miami) understood how to issue a PR. Please disregard!